src/ucp/am/eager_single.c

```diff
@@ -227,7 +341,7 @@ static void ucp_proto_eager_am_zcopy_add_footer(ucp_request_t *req,
 }
 
 static ucs_status_t
-ucp_proto_eager_am_zcopy_single_send_func(ucp_request_t *req,
+ucp_proto_am_eager_single_zcopy_send_func(ucp_request_t *req,
```

Comment:
IMO for consistency the functions in this file should start with ucp_am_... (but if it's too big change can be in another PR)

Reply:
Ok. I will open a separate PR to update names right after merging https://github.com/openucx/ucx/pull/7988 and https://github.com/openucx/ucx/pull/8019.